### PR TITLE
Picklist to Checkbox Automation

### PR DESCRIPTION
**Pull request by schlossmj**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  Store Template Record Id              ${template}
+  Create Gift Entry Batch               ${template}  ${template} Batch
+  Current Page Should Be                Form       Gift Entry   title=Gift Entry Form
+  Save Current Record ID For Deletion   ${NS}DataImportBatch__c
+  Fill Gift Entry Form
+  ...                                   Data Import: Donation Donor=Contact1
+  ...                                   Data Import: Contact1 Imported=${CONTACT}[Name]
+  Verify Field Default Value             
+  ...                                   Contact 1: Contact1 Checkbox To Picklist=True
+  ...                                   Contact 1: Contact1 Checkbox To Checkbox=
```
Comment (schlossmj) [inline robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot:59]: This is to verify that the checkbox value is false - the test fails when the value is actually 'false'.

File: robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+...             Open Test Browser
+...             Setup Test Data
+...             API Check And Enable Gift Entry
+Suite Teardown  Run Keywords
+...             Delete Session Records
+...             Capture Screenshot and Delete Records and Close Browser
+
+*** Variables ***
+
+
```
Comment (skristem) [inline robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot:16]: u dont have to add this if you are not using any variables

File: robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  Click Link                            Templates
+  Click Gift Entry Button               Create Template
+  Current Page Should Be                Template                    GE_Gift_Entry
+  Enter Value in Field
+  ...                                   Template Name=${template}
+  ...                                   Description=This is created by automation script 
+  Click Gift Entry Button               Next: Form Fields
+  Perform Action on Object Field        select   Contact 1  Contact1 Checkbox To Checkbox
+  Perform Action on Object Field        select   Contact 1  Contact1 Checkbox To Picklist
+  Page Should Not Contain Element       //lightning-input[@data-qa-locator="checkbox Required Contact 1: Contact1 Checkbox To Checkbox"]
```
Comment (skristem) [inline robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot:45]: please move this locator to locators file and use locator manager strategy in the doc. We also have some Gift Entry tests thats using this strategy already https://salesforce.quip.com/oyjJAcFaWtm8

File: robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot (hunk 2 above)
Comment (schlossmj) [reply robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot]: Removed.

File: robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot (hunk 3 above)
Comment (schlossmj) [reply robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot]: Done!

**Review by rjanjanam — APPROVED** [review]
Nice work

**Pull request merged**